acceptance/suites/tests/00_smoke/validate-syslog-logback.rb

```diff
@@ -50,15 +50,17 @@
   on(master, "service #{service_name} status")
 end
 
-
 step 'Backup logback'
   on(master, "mv #{logback_path} #{logback_backup}")
 
-step 'Modify logback configuration'
+step 'Modify logback configuration' do
   create_remote_file(master, logback_path, logback_config)
+  on(master, "chmod +r #{logback_path}")
+end
 
 step 'Restart puppetserver'
   on(master, "service #{service_name} restart")
```

Comment:
I'd been thinking we wanted to use `puppet("resource service #{service} ensure=stopped")` ... `ensure=running` to handle the restarts instead of doing the lower-level `service` command because it allows for the tests to be run under a wider swath of OSes in a more compatible way.  Is that not the case?  Not sure if that approach would also help avoiding the extra `sleep` below.  If it does, I'd definitely be more in favor of that.  The `sleep` seems more risky to me.